src/App.tsx

```diff
@@ -1,61 +1,111 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/todo';
+import { getUserById } from './services/userService';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodo: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
Ensure that the `getUserById` function is correctly implemented and returns the expected user object. If `getUserById` returns `undefined` for any `userId`, it could lead to issues when rendering the todos.